index.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path='typings/tsd.d.ts' />
+/// <reference path='typings/definitions.d.ts' />
```

Comment:
Well, I don't really understand the two choices you mentioned, but my preference would be a centralize type definition file that blends the `tsd` types and our overrides, and every `ts` file reference the same file. I am not too strong about the `path` stability issue, since refactoring the path should be part of directory restructuring?